### PR TITLE
wrapped VM Output for API responses

### PR DESCRIPTION
**Pull request by bogdan-rosianu**

### CODE REVIEW

File: api/mock/facade.go [hunk 1]
```diff
@@ -127,7 +127,7 @@ func (f *Facade) SendUserFunds(receiver string, value *big.Int) error {
 }
 
 // ExecuteSCQuery -
-func (f *Facade) ExecuteSCQuery(query *data.SCQuery) (*vmcommon.VMOutput, error) {
+func (f *Facade) ExecuteSCQuery(query *data.SCQuery) (*vm.VMOutputApi, error) {
```
Comment (sasurobert) [inline api/mock/facade.go:130]: this is not VMOutputApi - it is a data structure. Suggestion to rename to WrappedVMOutput

File: go.sum [hunk 2]
```diff
@@ -9,25 +9,31 @@ dmitri.shuralyov.com/state v0.0.0-20180228185332-28bcc343414c/go.mod h1:0PRwlb0D
 git.apache.org/thrift.git v0.0.0-20180902110319-2566ecd5d999/go.mod h1:fPE2ZNJGynbRyZ4dJvy6G277gSllfV2HJqblrnkyeyg=
 github.com/AndreasBriese/bbloom v0.0.0-20180913140656-343706a395b7/go.mod h1:bOvUY6CB00SOBii9/FifXqc0awNKxLFCL/+pkDPuyl8=
 github.com/AndreasBriese/bbloom v0.0.0-20190306092124-e2d15f34fcf9/go.mod h1:bOvUY6CB00SOBii9/FifXqc0awNKxLFCL/+pkDPuyl8=
+github.com/BurntSushi/toml v0.3.1 h1:WXkYYl6Yr3qBf1K79EBnL4mak0OimBfB0XUf9Vl28OQ=
```
Comment (sasurobert) [inline go.sum:12]: go mod tidy

**Review by sasurobert — CHANGES_REQUESTED** [review]

File: api/mock/facade.go (hunk 1 above)
Comment (iulianpascalau) [reply api/mock/facade.go:130]: It somehow resembles with the other wrapping data structures used in API package, hence the suffix: `Api`

**Review by iulianpascalau — APPROVED** [review]

File: api/mock/facade.go (hunk 1 above)
Comment (bogdan-rosianu) [reply api/mock/facade.go:130]: Yes. In addition to what Iulian said, we have already merged the corresponding PR on elrond-go. Can remain like this

File: go.sum (hunk 2 above)
Comment (bogdan-rosianu) [reply go.sum:12]: already done. Tried to remove the go mod and run 'go mod vendor' and 'go mod tidy' but I got the same output.

**Review by sasurobert — APPROVED** [review]

**Pull request merged**